do-dont-ns-card (#529)
centrica-engineering/nucleus-docs: src/content/docs/components/ns-card.mdx

## Guidance

:::do
- Use them in a triple-card layout.
- Use to direct customers to the actions.
- Embolden text in the paragraph that is important.
- Make the [CTA](/components/ns-cta) (Call To Action) wording unique, stating the action.
- Use [section cards](#section) to navigate users to primary actions.

> - Consider using decoration in the panel to add emphasis to the cards.

Reviewer: A link may be useful here:

```suggestion
- Consider adding a [decoration](/components/ns-panel/#specification-attribute-decoration) to the panel to emphasise the cards.
```

:::

:::dont

> - Use the `text` type CTA in the section card.

Reviewer: I think this can be deleted, It's not relevant any more. This was included when the CTA overlapped the boundary of the card in the previous brand.
```suggestion
```
